src/main/java/edu/iis/powp/command/gui/CommandEditWindow.java

```diff
@@ -35,15 +39,15 @@
  */
 public class CommandEditWindow extends JFrame implements WindowComponent {
 
-	private PlotterCommandManager commandManager;
-
+	private static final long serialVersionUID = 9204679248304669948L;
+	
+	private final PlotterCommandManager commandManager;
+	
 	private JList list;
-	private int subCommandCounter = 0;
 	private DefaultListModel model;
```

Comment:
The declared type could be AbstractListModel.
GUI is still not definitely separated from the logic (all in one file). Is it possible that extracting logic to other class and sharing the model would make it possible?